.github/workflows/ci.yml

```
@@ -101,28 +106,33 @@
-
+          
       - name: Build Development
         run: |
           npm ci
           eas build --platform all --profile development --non-interactive --no-wait
         env:
           EXPO_PUBLIC_SENTRY_DSN: ${{ secrets.EXPO_PUBLIC_SENTRY_DSN }}
-
+          
       - name: Store build artifacts
         uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)